### PR TITLE
testInvPolyArcLength3

### PR DESCRIPTION
**Pull request by loicmarx**
The function InvPolyArcLength3 has been tested.

### CODE REVIEW

File: test/suite/testInvPolyArcLength3.m [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+% compare the reference data and the new generated data
+assert(isequal(t_ref, t_new)); 
+
+% capture the warning message 
+warningMessage = 'invPolyArcLength3: given arcLength is negative! Forcing t=0. This is wrong but might be approximatly okay for the use case! Check carefully!';
+assert(verifyFunctionWarning('invPolyArcLength3', warningMessage, 'inputs', {polyCoeff, arcLength}));
+
+%% setup the reference data to test if ~arcLength(i)<0
+polyCoeff;
```
Comment (laurentheirendt) [inline test/suite/testInvPolyArcLength3.m]: What does this mean?

File: test/suite/testInvPolyArcLength3.m [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+arcLength_2 = 10; 
+t_ref2 = 0; 
+
+% generate the new output 
+t_new2 = invPolyArcLength3(polyCoeff, arcLength_2);
```
Comment (laurentheirendt) [inline test/suite/testInvPolyArcLength3.m]: Why are the variables named `2`?

File: test/suite/testInvPolyArcLength3.m [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+%     - Loic Marx, May 2019
+
+% save the current path
+currentDir = pwd;
+
+% initialize the test
+fileDir = fileparts(which(mfilename));
+cd(fileDir);
+
+%% setup the reference data to capture the warning message
```
Comment (laurentheirendt) [inline test/suite/testInvPolyArcLength3.m]: No need for double `%` signs.

**Review by laurentheirendt — CHANGES_REQUESTED** [review]

File: test/suite/testInvPolyArcLength3.m [hunk 4]
```diff
@@ -0,0 +1,43 @@
+% The PaCER Toolbox: testInvPolyArcLength3.m
+%
+% Purpose:
+%     - test the invPolyArcLength3 function
+%
+% Author:
+%     - Loic Marx, May 2019
+
```
Comment (laurentheirendt) [inline test/suite/testInvPolyArcLength3.m:8]: Is there a need here for the global variables?

File: test/suite/testInvPolyArcLength3.m (hunk 1 above)
Comment (loicmarx) [reply test/suite/testInvPolyArcLength3.m]: It tests the condition if arc length is not a negative value

File: test/suite/testInvPolyArcLength3.m (hunk 2 above)
Comment (loicmarx) [reply test/suite/testInvPolyArcLength3.m]: It actually tests the second condition

File: test/suite/testInvPolyArcLength3.m (hunk 1 above)
Comment (laurentheirendt) [reply test/suite/testInvPolyArcLength3.m]: I am actually talking about the `polyCoeff;` line ...

File: test/suite/testInvPolyArcLength3.m (hunk 2 above)
Comment (laurentheirendt) [reply test/suite/testInvPolyArcLength3.m]: I think there is no real need for adding `2`

File: test/suite/testInvPolyArcLength3.m (hunk 4 above)
Comment (loicmarx) [reply test/suite/testInvPolyArcLength3.m:8]: There is no need for the global variables here while reference data has been hard-coded

File: test/suite/testInvPolyArcLength3.m (hunk 1 above)
Comment (loicmarx) [reply test/suite/testInvPolyArcLength3.m]: Polycoeff  represents the coefficients of  a third degree polynomial, should I add a note in the test file ?

File: test/suite/testInvPolyArcLength3.m (hunk 4 above)
Comment (laurentheirendt) [reply test/suite/testInvPolyArcLength3.m:8]: Ok great! Maybe add a note.

File: test/suite/testInvPolyArcLength3.m (hunk 1 above)
Comment (laurentheirendt) [reply test/suite/testInvPolyArcLength3.m]: But what does this command do?

**Review by laurentheirendt — APPROVED** [review]

**Comment by laurentheirendt** [comment]
Thanks @loicmarx 👍

**Pull request merged**